pepper-apis/src/main/java/org/broadinstitute/ddp/housekeeping/handler/EmailNotificationHandler.java

```diff
@@ -283,6 +283,7 @@ boolean messageShouldBeIgnored(Handle apisHandle, String studyGuid, String parti
                             apisHandle);
                     pdfBucketService.sendPdfToBucket(blobName, pdfStream);
                     LOG.info("Uploaded pdf to bucket {} with filename {}", pdfBucketService.getBucketName(), blobName);
+                    pdfStream = pdfBucketService.getPdfFromBucket(blobName).orElse(null);
```

Comment:
pdfStream already consumed by pdfBucketService call above, hence get it again from bucket.